fix "multiple definition" errors by adding -fcommon compiler options,
see https://www.gnu.org/software/gcc/gcc-10/porting_to.html

author: Kenneth Hoste (HPC-UGent)

diff -ruN HWxtest.orig/MD5 HWxtest/MD5
--- HWxtest.orig/MD5	2020-11-08 21:15:03.612622261 +0100
+++ HWxtest/MD5	2020-11-08 21:13:24.278691443 +0100
@@ -40,6 +40,7 @@
 4657a2011dc5363ed1cfd1d29a7040d5 *src/HWmtest.c
 08ec52ea231752fbb2bb11a529107806 *src/HWxcount.c
 23598383650c211dd18d54bded68b9bd *src/HWxtest.c
+8b6d9aec8b8cef60d1b245e0f290bfed *src/Makevars
 886a6d855fb0b59e565b919a1ff69eb0 *vignettes/HWxtest.Rmd
 fcf857c5f98963f740c5c2f95e33ccc9 *vignettes/HWxtest.md
 b93f768037a75ec9edd60ca89fd4c1db *vignettes/bibHW.txt
diff -ruN HWxtest.orig/src/Makevars HWxtest/src/Makevars
--- HWxtest.orig/src/Makevars	1970-01-01 01:00:00.000000000 +0100
+++ HWxtest/src/Makevars	2020-11-08 21:15:51.340615023 +0100
@@ -0,0 +1 @@
+PKG_CFLAGS = -fcommon
